stories/organisms/components/filePicker/FilePicker.stories.mdx

```diff
@@ -97,6 +97,36 @@ It is a combination of the [FileInput](/docs/atoms-fileinput--overview) and the
 
 <ArgsTable story="Overview" />
 
+## Clear all
+
+Need to delete all uploaded files at once?  
+We thought it would be handy to offer you a turnkey way to do it!  
+For that, you can enable the `showClearAll` property (by giving it `true`) and that's it!
+The displayed text is themed and planned to fit harmoniously above the list.
+By default the `showClearAll` value is `false`.
```

Comment:
Just small remark but values should be in bold as specified by the [wiki](https://github.com/okp4/ui/wiki/Create-A-Story).

I would also like to take the opportunity to suggest some changes and adding description about the `showClearAll` internationalisation.

```suggestion
For that, you only have to pass `showClearAll` to the filePicker and that's it!
The displayed text is themed and internationalized (english and french) and fits harmoniously above the list.
By default the `showClearAll` value is **false**.
```

Reply:
Ok
Will be done in this PR : https://github.com/okp4/ui/pull/534